Dockerfile.postgres

```diff
@@ -0,0 +1,10 @@
+FROM postgres:latest
+
+# Copy the SQL scripts into a temporary directory
+COPY ./database/schema/*.sql /docker-entrypoint-initdb.d/
+
+# Grant permissions to the scripts
+RUN chmod +x /docker-entrypoint-initdb.d/*.sql
```

Comment:
what is this script and form where come from?

Reply:
this create schema of the database on postgres

Reply:
so write it in the comment